Simplified/RemoteHTMLViewController.swift

```diff
@@ -79,7 +79,7 @@ final class RemoteHTMLViewController: UIViewController, WKNavigationDelegate {
         decisionHandler(.cancel)
       } else {
         if #available(iOS 10.0, *) {
-          UIApplication.shared.open(url, options: [:], completionHandler: nil)
+          UIApplication.shared.open(url, options: convertToUIApplicationOpenExternalURLOptionsKeyDictionary([:]), completionHandler: nil)
```

Comment:
Wow I'm not sure what the migrator had in mind here 😄 
I checked the method docs for UIApplication [https://developer.apple.com/documentation/uikit/uiapplication/1648685-open](here) and it looks like both `options` and `completionHandler` have the same default values, so what SHOULD work is:
```
UIApplication.shared.open(url)
```
Let me know if that works.